lib/member_formatter.rb

```diff
@@ -12,7 +18,11 @@ def initialize(member)
   end
 
   field :name do
-    person[:name]
+    names[0].tidy
+  end
+
+  field :other_name do
```

Comment:
This should probably be `:alternate_names`, so that CSV-to-Popolo will add a `note`, rather than leaving it bare.